src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -189,8 +189,12 @@ namespace winrt::TerminalApp::implementation
             Icon(_lastIconPath);
             TabViewItem().IconSource(IconPathConverter::IconSourceMUX(_lastIconPath));
 
-            // Update SwitchToTab command's icon
-            SwitchToTabCommand().Icon(_lastIconPath);
+            // TODO CARLOS: We crash without this check :(

```

Comment:
These two `TODO CARLOS` are part of why this is still in draft. `SwitchToTabCommand` is null for some reason. This is most likely related to the other blocking issue: the tabs have no content inside of them.

If I can get a fresh pair of eyes on this issue, this'll be ready for review/merge.